app/src/test/kotlin/org/imperial/mrc/modelserver/integration/ModelserverApplicationTests.kt

```diff
@@ -16,8 +17,10 @@ class ModelserverApplicationTests(@Autowired val restTemplate: TestRestTemplate)
 	}
 
 	@Test
-	fun `Assert blog page title, content and status code`() {
+	fun `Assert redirects to login page`() {
```

Comment:
I'm not really sure quite how these integration tests need to work, can we talk about it on Monday?